src/app/profile/view.nim

```diff
@@ -197,12 +198,18 @@ QtObject:
     self.profile.setSendUserStatus(sendUserStatus)
     self.status.saveSetting(Setting.SendUserStatus, sendUserStatus)
 
-  proc showOSNotification*(self: ProfileView, title: string, message: string, 
+  proc showOSNotification*(self: ProfileView, title: string, message: string,
     notificationType: int, useOSNotifications: bool) {.slot.} =
 
     let details = OsNotificationDetails(
       notificationType: notificationType.OsNotificationType
     )
 
-    self.appService.osNotificationService.showNotification(title, message, 
-    details, useOSNotifications)
\ No newline at end of file
+    self.appService.osNotificationService.showNotification(title, message,
+    details, useOSNotifications)
+
+  proc logDir*(self: ProfileView): string {.slot.} =
+    if defined(windows):
```

Comment:
all this magic is already implemented in Qt: https://doc.qt.io/qt-5/qurl.html#fromLocalFile

Reply:
https://github.com/status-im/status-desktop/pull/3556/files#r709089357

Reply:
@michaelsbradleyjr I updated my PR exposing the local path of a file, see nimqml and dotherside

Reply:
My goal here (could be wrong goal, depends what we want) is for Qt to trigger the OS' default filesystem browser to open the *directory* re: `file://` URL constructed from `constants.nim`'s `LOGDIR`.

For non-Windows OS, that involves simply prepending `file://` to an absolute path, and on Windows [it involves](https://en.wikipedia.org/wiki/File_URI_scheme#Windows) prepending `file:///` (note triple vs. double `/`) and converting `\` to `/` in an absolute path.

I'm happy to look at `fromLocalFile` recommended by @pinebit and what @alaibe has been working on, but are those things relevant if the goal is triggering the opening of a directory with the OS' default filesystem browser?

Reply:
@pinebit when I first looked at [`url_toLocalFile`](https://github.com/status-im/status-desktop/pull/3556/files#diff-52d5dd4006f6b2b01f8e487717d1dae12d077834e20ab7afdae5367c71a90f74R495) and its implementation, I was a little confused, but then I figured out what you were recommending (I didn't pay enough attention to your linking to `fromLocalFile`).

See: https://github.com/status-im/nimqml/pull/34#issuecomment-920315845 and https://github.com/status-im/dotherside/pull/52#issuecomment-920297210.

With the changes I made/suggested in place, I'll change `proc logDir` to:
```nim
proc logDir*(self: ProfileView): string {.slot.} =
  url_fromLocalFile(constants.LOGDIR)
```

Reply:
Change made in a commit I just pushed.

Reply:
Great job @michaelsbradleyjr! 
This is exactly what I expected.